src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Controllers/AuthenticationControllerTests.cs

```diff
@@ -792,13 +1121,30 @@ public async Task AuthenticateStudioToken_InvalidToken_ReturnsUnauthorized()
             Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
         }
 
-        private HttpClient GetTestClient(ISblCookieDecryptionService cookieDecryptionService, IUserProfileService userProfileService)
+        private HttpClient GetTestClient(ISblCookieDecryptionService cookieDecryptionService, IUserProfileService userProfileService, bool enableOidc = false, bool oidcDefault = false, string defaultOidc = "altinn")
```

Comment:
None of the tests overrides the default value for `defaultOidc`. Is that a forgotten scenario?

Reply:
added